developer-docs-site/docs/community/site-updates.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Update Aptos.dev"
+slug: "site-updates"
+---
+
+# Update Aptos.dev
+
+As an open source project, Aptos needs your knowledge to grow. Follow the instructions on this page to update [Aptos.dev](https://aptos.dev/), the developer website for the Aptos blockchain. Every contributor to Aptos.dev is listed as an *author* on the pages they edit and update. See the *Authors* list at the bottom of any page for an example.
+
+See the [Aptos Docs](https://github.com/orgs/aptos-labs/projects/14/views/1) project for open issues by status. See detailed instructions for making updates below.
+
+## tl;dr
+
+Simply click **Edit this page** at the bottom of any location to go to the source and trigger editing there. The contents are in [Markdown](https://www.markdownguide.org/basic-syntax/) format. You may then edit in browser and use the *Preview* function to view your changes.
+
+Here are the basic steps for editing in your web browser:
+
+1. Click **Edit this page** at the bottom to get started.
+2. Modify and add source Markdown files in the [developer-docs-site](https://github.com/aptos-labs/aptos-core/tree/main/developer-docs-site) directory.
+3. See your changes in Netlify (by swapping `prnumber` in):
```

Comment:
I don't think this is available to the general public, so they should verify locally or we can make this instruction for reviewers.

Reply:
I tested in Incognito. For better or worse, it is available. I mention the ability to verify locally elsewhere.